### PR TITLE
[11.0][ADD] web_refresher

### PR DESCRIPTION
**Pull request by josueBulle**
Adds a button next to the pager in list views. It updates the displayed page of the list.

refresh

### CODE REVIEW

File: web_refresher/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+##############################################################################
```
Comment (tarteo) [inline web_refresher/__manifest__.py]: Can you remove the banner, please.

**Review by tarteo — CHANGES_REQUESTED** [review]
One remark otherwise :+1: Tested it on Runbot works perfectly

**Review by tarteo — APPROVED** [review]
Thanks!

**Review by ecino — APPROVED** [review]
:+1:

**Pull request merged**